packages/core-react/src/components/Select/index.js

```diff
@@ -10,34 +17,88 @@ export default function Select({
   disabled,
   placeholder,
   label,
-  children
+  active = false,
+  error = false,
+  prepend = false,
+  iconPosition,
+  icon,
+  value = '',
+  children,
+  onChange,
+  ...rest
 }) {
+  const [activeClass, setActiveState] = React.useState(active);
+  const [currValue, setValue] = React.useState(value);
+
+  const handleFocus = () => {
+    setActiveState(true);
+  };
+
+  const handleBlur = () => {
+    setActiveState(false);
+  };
+
+  const handleChange = event => {
+    setValue(event.target.value);
+    onChange(event);
```

Comment:
I'd write this defensively as `onChange && onChange(event);` so it won't throw an error when `onChange` isn't specified.